installing/installing_aws/installing-aws-secret-region.adoc

```diff
@@ -0,0 +1,72 @@
+[id="installing-aws-secret-region"]
```

Comment:
Needs `:content-type: ASSEMBLY`

Reply:
Done. Thank you.

Reply:
Sorry, I steered you wrong here. 

Can you change `:content-type: ASSEMBLY` to `:_content-type: ASSEMBLY`

I think we're good to go after that. Thanks!